src/coreclr/nativeaot/System.Private.CoreLib/src/System/Array.NativeAot.cs

```diff
@@ -93,6 +93,7 @@ private static unsafe Array InternalCreate(RuntimeType elementType, int rank, in
             }
         }
 
+#pragma warning disable CA1859
```

Comment:
Why is this suppressed? I've avoided enabling this rule yet because there are still bugs leading to false positives, e.g. https://github.com/dotnet/roslyn-analyzers/issues/6440.  If this is such a false positive, please include a link to the associated issue.  This particular analyzer is one I'd expect to have effectively zero false positives, the only case I'm aware of where there would be a need to suppress in cases where we use ifdefs that influence different types.

Reply:
Yes, it's a bug in analyzer just didn't have time to extract all false positives into bug reports.

Reply:
Let's please do so before this is merged.

Reply:
Totally, that's why it's still a draft